plugins/connection/httpapi.py

```diff
@@ -97,6 +97,17 @@
     default: true
     vars:
     - name: ansible_httpapi_use_proxy
+  ciphers:
+    description:
+      - SSL/TLS Ciphers to use for requests
+      - 'When a list is provided, all ciphers are joined in order with C(:)'
+      - See the L(OpenSSL Cipher List Format,https://www.openssl.org/docs/manmaster/man1/openssl-ciphers.html#CIPHER-LIST-FORMAT)
+        for more details.
+      - The available ciphers is dependent on the Python and OpenSSL/LibreSSL versions
```

Comment:
```suggestion
      - The available ciphers is dependent on the Python and OpenSSL/LibreSSL versions
      - This option will have no effect on ansible-core<2.14.
```

Reply:
Updated in https://github.com/ansible-collections/ansible.netcommon/pull/494/commits/73e9010179949b941481b40fcc9af1af91a5b0d3